src/execution/isolate.cc

```diff
@@ -1492,6 +1509,27 @@ void Isolate::InvokeApiInterruptCallbacks() {
   }
 }
 
+void Isolate::RecordReplayInvokeApiInterruptCallbacksAtProgress() {
+  if (!recordreplay::IsRecordingOrReplaying("interrupts")) {
+    return;
+  }
+
+  if (recordreplay::IsRecording() && IsMainThread()) {
+    while (true) {
+      InterruptEntry entry;
+      {
+        ExecutionAccess access(this);
+        if (api_interrupts_queue_.empty()) return;
```

Comment:
Nit: Syntax, braces. That return is snuck away in there and it's hard to easily tell where the loop break.

Reply:
Yeah, this was copy-pasted from InvokeApiInterruptCallbacks, chromium/v8's code style can make things a little tricky to follow sometimes.